src/Client/Style.fs

```diff
@@ -18,7 +18,7 @@ let viewLink page description =
       Href (toHash page) ]
     [ unbox description]
 
-let internal centerStyle direction =
```

Comment:
Given the "light touch" nature of the coding I don't think there's much point in the sample using accessibility controls.